src/test/java/seedu/linkedout/logic/parser/SearchCommandParserTest.java

```diff
@@ -49,22 +58,28 @@ public void parse_validArgs_returnsSearchCommand() {
                 new SearchCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
         SearchCommand expectedJobSearchCommand =
                 new SearchCommand(new JobContainsKeywordsPredicate(Arrays.asList("Software", "Engineer")));
+        SearchCommand expectedRoundSearchCommand =
+                new SearchCommand(new RoundContainsKeywordsPredicate(Arrays.asList("Technical", "Interview")));
 
         // no leading and trailing whitespaces
         assertParseSuccess(parser, " n/Alice Bob", expectedNameSearchCommand);
         assertParseSuccess(parser, " j/Software Engineer", expectedJobSearchCommand);
+        assertParseSuccess(parser, " r/Technical Interview", expectedRoundSearchCommand);
 
         // multiple whitespaces between keywords
         assertParseSuccess(parser, " n/Alice \n \t Bob  \t", expectedNameSearchCommand);
         assertParseSuccess(parser, " j/Software \n \t Engineer  \t", expectedJobSearchCommand);
+        assertParseSuccess(parser, " r/Technical \n \t Interview            \t", expectedRoundSearchCommand);
 
         //leading whitespace after prefix
         assertParseSuccess(parser, " n/\nAlice \n \t Bob  \t", expectedNameSearchCommand);
         assertParseSuccess(parser, " j/                  Software \n \t Engineer  \t", expectedJobSearchCommand);
+        assertParseSuccess(parser, " r/        Technical     \n         Interview  ", expectedRoundSearchCommand);
 
```

Comment:
Same as above, possible to include multiple keywords Success and failures

Reply:
Noted. As discussed in the meeting, @goel-a will add them in when he does his task!